roles/install_telegraf/defaults/main.yml

```diff
@@ -133,6 +141,13 @@ telegraf_conf_inputs_content: |
       "dss.*.server.dku.notifications.publish.*             measurement.dss_id.measurement.measurement.measurement.field.dimension notification=all",
     ]
   {% endif %}
+  {% endif %}
+telegraf_conf_inputs_with_secretstores_content: |
+  {% if telegraf_conf_use_legacy_graphite | bool is false %}
+  [[inputs.prometheus]]
+    urls = ["http://localhost:10000/public/api/metrics-prometheus"]
+    username = "{% raw %}@{{% endraw %}{{ telegraf_secret_store_name }}:dss_api_key{% raw %}}{% endraw %}"
```

Comment:
You could write rather `{{ '{' + telegraf_secret_store_name + ':dss_api_key}' }}`

Reply:
not relevant anymore as we opted out of telegraf secrets store